pom.xml

```diff
@@ -146,7 +201,7 @@
             <dependency>
```

Comment:
I think this should be as part of a <dependencyManagement> field like in java and spark projects